metadata-service/factories/src/main/java/com/linkedin/gms/factory/entity/EbeanServerFactory.java

```diff
@@ -22,12 +23,24 @@
   @DependsOn({"gmsEbeanServiceConfig"})
   @Nonnull
   protected EbeanServer createServer() {
-    ServerConfig serverConfig = applicationContext.getBean(ServerConfig.class);
-    // Make sure that the serverConfig includes the package that contains DAO's Ebean model.
-    if (!serverConfig.getPackages().contains(EBEAN_MODEL_PACKAGE)) {
-      serverConfig.getPackages().add(EBEAN_MODEL_PACKAGE);
+    try {
+      ServerConfig serverConfig = applicationContext.getBean(ServerConfig.class);
+
+      // Make sure that the serverConfig includes the package that contains DAO's Ebean model.
+      if (!serverConfig.getPackages().contains(EBEAN_MODEL_PACKAGE)) {
+        serverConfig.getPackages().add(EBEAN_MODEL_PACKAGE);
+      }
+
+      io.ebean.EbeanServer factory = io.ebean.EbeanServerFactory.create(serverConfig);
+
+      if (factory == null) {
+        throw new RuntimeException("Failed to create ebean server, EbeanServerFactory is null!");
```

Comment:
I don't think the factory itself is null, since we are calling create on the factory. EbeanServer is null seems more appropriate

Reply:
Also might want to just have "EbeanServer is null" bc I believe the catch phrase below will encapsulate it and add the "Failed to create ebean server"